shell/server/sandcats.js

```diff
@@ -63,21 +63,24 @@ const pingUdp = () => {
     }
   });
 
-  socket.send(message, 0, message.length, 8080, SANDCATS_HOSTNAME, (err) => {
-    if (err) {
-      console.error("Couldn't send UDP sandcats ping", err);
-    }
-  });
+  socket.bind({ address: process.env.BIND_IP }, () => {
```

Comment:
@kentonv I'm not sure. The docs don't make it clear: https://nodejs.org/api/dgram.html#dgram_socket_bind_options_callback

Given the preceding section in that link, it seems like we should be listening to the `error` event? Does that seem right to you?